src/main/java/Remy/command/Command.java

```diff
@@ -0,0 +1,26 @@
+package remy.command;
+import remy.task.TaskList;
+import remy.Ui;
+import remy.Storage;
+import remy.ChatbotException;
+
+/**
+ * A representation of a Command that parses user input and follows-up with the relevant action.
+ */
+public abstract class Command {
+
+    /**
+     * Whether the Task should trigger the end of the current Chatbot session.
+     * @return boolean indicating Exit status.
+     */
+    public abstract boolean isExit();
```

Comment:
This could possibly be a concrete method that returns `false` by default, and only overridden by `ExitCommand` to return `true`. This is so that not every subclass that extends `Command` have to override it, improving simplicity of your code.